tests/lcb/compiler/bytecode.lcb

```diff
@@ -0,0 +1,291 @@
+/*
+Copyright (C) 2016 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+module com.livecode.compiler.bytecode.tests
+
+public handler TestJump()
+   variable tTest as Boolean
+   put false into tTest
+   bytecode
+      Jump lAfterAssign
+      AssignConstant tTest, true
+   lAfterAssign:
+   end bytecode
```

Comment:
While we're getting _mega_ bikesheddy: it's confusing that the label is "fully" unindented.  Maybe it would be clearer if it was indented by one space beyond the indentation of the `bytecode`/`end bytecode`?